packages/themes/src/types/index.ts

```diff
@@ -6,10 +6,10 @@ export type Dictionary<T = any> = Record<string, T>;
 
 type ThemeStyleResolve<T extends ComponentNames, K> = ComponentType<T> extends K
   ? keyof ComponentType<T>[keyof K]
-  : never;
+  : string;
 
 export type ThemeStyleProperties<T extends ComponentNames> = {
-  colorScheme?: ThemeStyleResolve<T, { colorScheme: any }>;
+  colorScheme?: ThemeStyleResolve<T, { colorScheme: string }>;
```

Comment:
ini kenapa di isi `string` va? apakah semua `colorScheme` kita itu udah pasti string?, soalnya pertama kali gw liat colorScheme di button itu function sebelum di rubah di pr yang sekarang. (need better discussion)

Reply:
fixed in [ef2310f](https://github.com/yoru-ui/yoru-ui/pull/38/commits/ef2310f9453e547353231d10b09e850fb29d50a4)